tiatoolbox/wsicore/wsireader.py

```diff
@@ -1134,61 +1339,109 @@ def _info(self):
             mpp=None,
             raw=None,
         )
-
-        warnings.warn("Unknown scale (no objective_power or mpp).")
-
-        warnings.warn("Raw data is None.")
+        self._m_info = param
 
         return param
 
-    def read_rect(self, location, size, resolution=1.0, units="baseline"):
-        """Find parameters for optimal read."""
-        (
-            _,
-            level_location,
-            _,
-            post_read_scale,
-            baseline_read_size,
-        ) = self._find_read_rect_params(
+    def _find_params_from_baseline(self, location, baseline_read_size):
+        """Convert read parameters from (virtual) baseline coordinates.
+
+        Args:
+            location (tuple(int)): Location of the location to read in
+                (virtual) baseline coordinates.
+            baseline_read_size (tuple(int)): Size of the region to read
+                in (virtual) baseline coordinates.
+        """
```

Comment:
Please add a new line at the end.

Reply:
Fixed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/61/commits/5b0e0ecc1fcb9ce989d907483bef9a2b0b2c7aa5